openapi/components/schemas/StorefrontQuote.yaml

```
@@ -94,38 +94,75 @@ properties:
       - $ref: ./Contact/ContactObject.yaml
   invoicePreview:
     type: object
-    description: Preview of initial quote invoice.
+    description: Preview of the quote invoice.
     readOnly: true
     properties:
       currency:
         description: Currency of the invoice.
         allOf:
           - $ref: ./CurrencyCode.yaml
-      amount:
-        description: Amount of the invoice.
-        type: number
-        x-type: Money
-        format: double
-      subtotalAmount:
-        description: Subtotal amount of the invoice.
-        type: number
-        x-type: Money
-        format: double
-      discountAmount:
-        description: Discount amount that is applied to the invoice.
-        type: number
-        x-type: Money
-        format: double
-      shippingAmount:
-        description: Shipping amount that is applied to the invoice.
-        type: number
-        x-type: Money
-        format: double
-      taxAmount:
-        description: Tax amount that is applied to the invoice.
-        type: number
-        x-type: Money
-        format: double
+      initialAmounts:
+        type: object
+        description: Total amounts of the initial invoice.
+        properties:
+          amount:
+            description: Amount of the invoice.
+            type: number
+            x-type: Money
+            format: double
+          subtotalAmount:
+            description: Subtotal amount of the invoice.
+            type: number
+            x-type: Money
+            format: double
+          discountAmount:
+            description: Discount amount that is applied to the invoice.
+            type: number
+            x-type: Money
+            format: double
+          shippingAmount:
+            description: Shipping amount that is applied to the invoice.
+            type: number
+            x-type: Money
+            format: double
+          taxAmount:
+            description: Tax amount that is applied to the invoice.
+            type: number
+            x-type: Money
+            format: double
+      recurringAmounts:
+        type: object
+        description: |-
+          Total amounts of the recurring invoice.
+          This includes recurring items only.
+          Can be `null` if the quote doesn't have recurring items.
```

suggestion: if possible, it's good to describe scenario before the result
```suggestion
          If the quote does not have recurring items, the value of this field is `null`.
```